packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -619,6 +619,7 @@ export class DeltaManager
             mode: connection.mode,
             serviceConfiguration: connection.serviceConfiguration,
             version: connection.version,
+            // timestamp: connection.timestamp,
```

Comment:
I found this function called in DeltaManager.connect, so not sure if I should add `timestamp` here as well, but that means I will need to add it to `IDocumentDeltaConnection` in driver-definitions